packages/backend/src/external/subgraph/types.ts

```diff
@@ -0,0 +1,13 @@
+export interface BoothCreated {
+  id: string;
+  blockNumber: string;
+  blockTimestamp: string;
+  boothAddress: string;
+  owner: string;
+  paymentOption: number;
```

Comment:
in contract,
0 - NativeCurrency
1 - ERC20Token

so you can make it enum type